ts/llm_launcher.py

```diff
@@ -0,0 +1,162 @@
+import argparse
+import contextlib
+import shutil
+from pathlib import Path
+from signal import pause
+
+import torch
+import yaml
+from model_archiver import ModelArchiverConfig
+from model_archiver.model_packaging import generate_model_archive
+
+from ts.launcher import start, stop
+
+
+def get_model_config(args):
+    download_dir = getattr(args, "vllm_engine.download_dir")
+    download_dir = (
+        Path(download_dir).resolve().as_posix() if download_dir else download_dir
+    )
+
+    model_config = {
+        "minWorkers": 1,
+        "maxWorkers": 1,
+        "batchSize": 1,
+        "maxBatchDelay": 100,
+        "responseTimeout": 1200,
+        "deviceType": "gpu",
+        "asyncCommunication": True,
+        "parallelLevel": torch.cuda.device_count() if torch.cuda.is_available else 1,
+        "handler": {
+            "model_path": args.model_id,
+            "vllm_engine_config": {
```

Comment:
Same as above, Can we make this generic to support other engines?

Reply:
see above, when we can add an --engine command in follow up pr with default vllm we can add support for other engines without changing the current interface

Reply:
Cool. That makes sense